src/tokenizer/index.js

```diff
@@ -103,11 +109,13 @@ var TypeaheadTokenizer = React.createClass({
     tokenClasses[this.props.customClasses.token] = !!this.props.customClasses.token;
     var classList = classNames(tokenClasses);
     var result = this.state.selected.map(function(selected) {
-      var displayString = this.props.displayOption(selected);
+      var value = Accessor.valueForOption(this.props.formInputOption, selected);
```

Comment:
What do you think about making it so that the default value for the token is the display string if no formInputOption is set?